content/learn/additional-resource.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: "Additional Resources"
+---
+
+# Additional Resources
+
+<Intro>
+  We hope these pages will help you submit effective testimony. You may want to
```

Comment:
Once we have translations, we'll need to add indirection for these content strings. From [this](https://react.i18next.com/guides/quick-start), this section would look like:

```js
<Intro>
  t('additional_resources.intro')
</Intro>
```

So the mdx file is no longer the source of truth for content. I'm wondering what mdx gives us in this case that better-factored tsx wouldn't.

Reply:
I had time to go over the i18next docs last night and, if that's the direction we want to take, we should drop MDX support in favor of [i18next-http-backend](https://github.com/i18next/i18next-http-backend). Here is an [article](https://locize.com/blog/react-i18next/#separate) spelling out  how to create a `locales` directory with each translation.

Should I go ahead and swap the approach on this PR? I have some time later today.

Reply:
@webrgp I am not aware of any i18n approach that doesn't involve this sort of indirection, though I am totally open to such options. If not, please go ahead and integrate i18next. Created [687](https://github.com/codeforboston/advocacy-maps/issues/687) to track.

react-i18next provides a package to simplify translations specifically for Next.js apps, but it relies on SSR to inject the translations. There _is_ a workaround that bends over backward to support static exports (see issue). Though, I think the simplest integration would be to load the translations on the client behind a loading state of the UI, but we may want to consider adding SSR down the road for a better UX. We originally did not use SSR for cost-savings/simplicity, but we are past that point now anyway and will be applying for funding soon. SSR is probably its own discussion, I'd love to hear your thoughts/experiences.